demo/roles/users/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
+- name: Create hpccloud user
+  become: yes
+  become_user: root
+  user: name=hpccloud shell=/bin/bash uid=1002
+
+- name: Give hpccloud passwordless sudo
+  become: yes
+  become_user: root
+  lineinfile: "dest=/etc/sudoers state=present regexp='^hpccloud' line='hpccloud ALL=(ALL) NOPASSWD: ALL'"
+
```

Comment:
The hpccloud user only get created when you deploy on the same machine.
So if you create a compute-node that does not have hpccloud/cumulus/girder/... Then you are missing that user.

So to be sure, I put that one back here. Since it does not harm the system to ensure a user hpccloud exist. And if not, then create it.